hack/test.sh

```diff
@@ -19,11 +19,12 @@ export KUBEVIRT_NUM_NODES=2
 export KUBEVIRT_PROVIDER=k8s-1.21
 make cluster-down
 make cluster-up
-wget https://dl.google.com/go/go1.16.7.linux-amd64.tar.gz
-tar -xzf go1.16.7.linux-amd64.tar.gz
-export GOROOT=$PWD/go
-export PATH=$GOROOT/bin:$PATH
-echo $PATH
+if [[ -v PROW_JOB_ID ]] ; then
```

Comment:
This part is used in at least 3 places. Maybe it is worth extracting into some common script. 
WDYT?

Reply:
Yes I was thinking about that, I should probably make it a function 'ensureGo' or something and put it in a common script. Since we are cleaning up stuff might as well do it right. Let me make that change.